tools/pythonpkg/duckdb_python.cpp

```diff
@@ -752,7 +752,7 @@ struct PandasScanFunction : public TableFunction {
 
 	static unique_ptr<FunctionOperatorData>
 	pandas_scan_parallel_init(ClientContext &context, const FunctionData *bind_data_, ParallelState *state,
-	                          vector<column_t> &column_ids, TableFilterSet *table_filters) {
+	                          vector<column_t> &column_ids, TableFilterSet *table_filters, TableFilterSet *zonemaps_checks) {
```

Comment:
Would prefer adding this to a struct to avoid needing to adjust all these function templates in the future, e.g. 

```
struct TableFilterCollection {
   TableFilterSet *table_filters;
   TableFilterSet *zonemaps_checks;
};```

Then in the future when more variables might need to be added relating to table filters they can simply be added to the struct.